### DISCUSSION TITLE
Is there a newer release of edu.ucar.cdm-core than 5.5.3?

### DISCUSSION THREAD

**Discussion by steffenlomholt** [Unidata/netcdf-java#1354] [Questions and Answers] [answered]
Is there a newer release of edu.ucar.cdm-core than 5.5.3?
I can't find any newer on Maven Central. 5.5.3 has numerous vulnerable dependencies.

**Comment by haileyajohnson** [comment, marked as answer]
Hi @steffenlomholt , we recommend users use our latest snapshot releases, which is available [here](https://downloads.unidata.ucar.edu/netcdf-java/), until our next release is available. The vulnerable dependencies are upgraded in the latest.

**Reply by steffenlomholt** [reply]
Thanks @haileyajohnson - it fixed my problem.
